src/createStore.js

```diff
@@ -71,6 +71,10 @@ export default function createStore(reducer, initialState, enhancer) {
    * @returns {any} The current state tree of your application.
    */
   function getState() {
+    if (isDispatching) {
+      throw new Error('Reducers may not access store state.')
```

Comment:
Maybe change it to something a little more helpful? I’m up for more descriptive error messages. e.g. `You may not call store.getState() while the reducer is executing. The reducer has already received the state as an argument, so pass it down from the top reducer instead of reading it from the store.`.